.github/workflows/main.yml

```diff
@@ -24,6 +24,7 @@ jobs:
       - name: Build
         shell: bash
         run: |
+          corepack enable
```

Comment:
It's strange that `corepack enable` is appearing as an added line &mdash; it [seems to already be present in `main.yml` (without this pull request)](https://github.com/joplin/website/blob/ff77cc705dfbe3b2ce3c0c814f33678b21c6db37/.github/workflows/main.yml#L27).